.github/workflows/cli-test-npm.yml

```diff
@@ -17,22 +17,40 @@ jobs:
         uses: actions/setup-node@v2
         with:
          node-version: ${{ matrix.node-version }}
+      - name: Check Out Repo
+        uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+         node-version: ${{ matrix.node-version }}
+      - name: Check for cached twilio installation 
+        run: | 
+         if ! command -v twilio > /dev/null; then twilio() { printf '%sCLI is not installed, all good. '; } else echo "CLI is installed already, Abort."; exit 1; fi
+         twilio --version
+      - name: Check Out Repo
```

Comment:
Why are we checking out the repo again here? We do this in Line 20 as well

Reply:
That can be removed, it's only needed once below. Taking it as a nit, I'll clean it up in the upcoming PR.